src/app/Command.h

```diff
@@ -48,18 +45,18 @@ namespace app {
 class Command
 {
 public:
-    enum CommandRoleId
+    enum class CommandRoleId
     {
-        kCommandSenderId  = 0,
-        kCommandHandlerId = 1,
+        SenderId  = 0,
+        HandlerId = 1,
```

Comment:
Maybe "Sender" and "Handler"?  Otherwise you get `CommandRoleId::SenderId` which is a bit repetitive...

Reply:
sure, will do it in next cleanup pr